tests/sentry/integrations/msteams/test_notifications.py

```diff
@@ -56,9 +56,7 @@ def _install_msteams_personal(self):
             name="Personal Installation",
             provider="msteams",
         )
-        self.idp = self.create_identity_provider(
-            integration=self.integration, type="msteams", external_id=self.tenant_id, config={}
-        )
+        self.idp = self.create_identity_provider(integration=self.integration)
```

Comment:
Prior to this branch, the kwargs other than `integration` were actually being ignored, so it's nice to tighten that up as well.